src/models/gen_embeddings.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
Keep notebooks out of your src files. If something essential for the source code is currently in a notebook, convert it to a script and put that here. You want all your source code, even stuff that only runs periodically, to be scripts. 

There's *a lot* of stigma in the software world against people using notebooks for shared code and production code, and you doing it could make you be perceived the wrong way. This may change in the future, but it hasn't yet for sure. Keep notebooks just for experiments and personal stuff. Until it becomes the accepted norm, try to avoid using them for anything else.